src/pr-info.ts

```diff
@@ -527,14 +519,18 @@ function downloadsToPopularityLevel(monthlyDownloads: number): PopularityLevel {
 }
 
 export async function getOwnersOfPackage(packageName: string, oid: string, fetchFile: typeof defaultFetchFile): Promise<string[] | null | Error> {
-    const indexDts = `${oid}:types/${packageName}/index.d.ts`;
-    const indexDtsContent = await fetchFile(indexDts, 10240); // grab at most 10k
-    if (indexDtsContent === undefined) return null;
-    let parsed: HeaderParser.Header;
+    const packageJson = `${oid}:types/${packageName}/package.json`;
+    const packageJsonContent = await fetchFile(packageJson, 10240); // grab at most 10k
```

Comment:
I suspect we can drop this, now that it's not a huge declaration file that has to be parsed.